src/main/java/com/google/devtools/build/lib/bazel/rules/java/java_stub_template.txt

```diff
@@ -312,6 +312,7 @@ function create_and_run_classpath_jar() {
       path="file:${path}" # e.g. "file:/usr/local/foo.jar"
     fi
 
+    path=$(sed "s/%/%25/g" <<< "${path}")
     path=$(sed "s/ /%20/g" <<< "${path}")
```

Comment:
This could be combined into a single sed invocation: `sed "s/ /%20/g; s/%/%25/g"`